Gopkg.lock

```diff
@@ -746,6 +746,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "325d0175a74db86d1fed482bddc8279459d63be9d5338d9a420722c24eea5534"
+  inputs-digest = "182946acde1cd53f9b8e6e3137ea55d1d14d7467a175ce261ddf30d74de3168e"
```

Comment:
i still don't quite understand why this changes every time even though we havent' changed gopkg.toml

Reply:
running `bin/dep hash-inputs` on master and this branch reveals two additional dependencies:
k8s.io/apimachinery/pkg/api/meta
k8s.io/apimachinery/pkg/runtime/schema

while both of these dependencies are already referenced via:
```
[[constraint]]
  name = "k8s.io/apimachinery"
  version = "kubernetes-1.9.1"
```
... we are still creating a new dependency in our code on those two packages.